man/ctags.1.rst.in

```diff
@@ -1842,6 +1842,10 @@ TMPDIR
 FILES
 -----
 
+$XDG_CONFIG_HOME/ctags/\*.ctags (on other than MSWindows)
```

Comment:
Makes no sense to exclude this on Windows. If `$XDG_CONFIG_HOME` is defined there, then the user clearly wants XDG compliance.